src/Fields/Relationships/BelongsToMany.php

```diff
@@ -417,10 +417,14 @@ protected function resolvePreview(): View|string
                         );
                 }
 
-                if ($this->inLineBadge) {
-                    return Badge::make((string) $value, 'primary')
-                        ->customAttributes(['class' => 'm-1'])
-                        ->render();
+                $badgeValue = value($this->inLineBadge, $item, (string) $value, $this);
+
+                if ($badgeValue) {
```

Comment:
если я правильно понял, Данил предложил сделать такую проверку
```suggestion
                if ($badgeValue !== false) {
```